src/Umbraco.Infrastructure/Security/UmbracoUserManager.cs

```diff
@@ -111,6 +111,54 @@ public string GeneratePassword()
             return password;
         }
 
+        /// <summary>
+        /// Generates a hashed password based on the default password hasher
+        /// No existing identity user is required and this does not validate the password
+        /// </summary>
+        /// <param name="password">The password to hash</param>
+        /// <returns>The hashed password</returns>
+        public string HashPassword(string password)
+        {
+            IPasswordHasher<TUser> passwordHasher = GetDefaultPasswordHasher(PasswordConfiguration);
+
+            string hashedPassword = passwordHasher.HashPassword(null, password);
+            return hashedPassword;
+        }
+
+        /// <summary>
+        /// Used to validate the password without an identity user
+        /// Validation code is based on the default ValidatePasswordAsync code
+        /// Should return <see cref="IdentityResult.Success"/> if validation is successful
+        /// </summary>
+        /// <param name="password">The password.</param>
+        /// <returns>A <see cref="IdentityResult"/> representing whether validation was successful.</returns>
+        public async Task<IdentityResult> ValidatePasswordAsync(string password)
```

Comment:
@Shazwazza needs confirming - is based on the default `ValidatePasswordAsync` code including the error code. I created this so our manager can carry out the validation we need based on the passwordvalidators.

Reply:
Seems good to me